src/components/InlineCheckbox/InlineCheckbox.js

```diff
@@ -97,6 +99,7 @@ export default class InlineCheckbox extends React.Component {
       onClick,
       onKeyDown,
       title = undefined,
+      forwardRef: ref,
```

Comment:
Should we use `forwardRef` since that's the method name from React? In the past we've done `innerRef`, I think.

Reply:
Makes sense, made a change.